notebooks/advanced_algorithms/Quantum_Principal_Component_Analysis.ipynb

```diff
@@ -0,0 +1,1514 @@
+{
```

Comment:
<p>Can we offer a reference for readers unfamiliar with covariance?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-braket-algorithm-library/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>